src/components/SidebarComponents/Layers.tsx

```diff
@@ -1,57 +1,21 @@
 import {
-  CalciteAccordion,
-  CalciteAccordionItem,
   CalciteBlock,
-  CalciteIcon,
-  CalciteLabel,
-  CalciteNotice,
-  CalciteSegmentedControl,
-  CalciteSegmentedControlItem,
-  CalciteSlider,
-} from '@esri/calcite-components-react'
+} from '@esri/calcite-components-react';
+import useCustomLayerList from '../../hooks/useCustomLayerList';
 
-function Layers() {
-  return (
-    // <CalciteAccordion scale='l' selection-mode='multi'>
-    //   {Array.from({ length: 5 }, (_, i) => (
-    //     <CalciteAccordionItem item-title={`Layer ${i + 1}`} key={i}>
-    //       <p>Content for Layer {i + 1}</p>
-    //     </CalciteAccordionItem>
-    //   ))}
-    // </CalciteAccordion>
+const Layers: React.FC = () => {
+  const layerList = useCustomLayerList();
```

Comment:
abstract out the layerlist creation to a custom hook at [/src/hooks/useCustomLayerList.tsx](https://github.com/UGS-GIO/geohaz-v2/pull/16/files#diff-d6c06cb6acf1c59f9aad2ed18001ea61902a54885646a33db9aa702a7becd30e)